src/evergreen.ts

```diff
@@ -80,7 +94,19 @@ export class client {
             username: username,
             password: password,
         };
-        this.postResource(callback, "login", params);
+        this.postResource(callback, "login", params, true);
+    }
+
+    /**
+     * Gets patches for a particular user
+     * 
+     * @param callback - function to process the response
+     * @param username - Evergreen user's username
+     * @returns nothing
+     */
+    public getPatches(callback: request.RequestCallback, username?: string) {
+        const url =  "json/patches/user/" + username;
```

Comment:
nit: I would name this "resource" since it's a path rather than a full URL

Reply:
Makes sense, fixed